lib/stitches/running_stitch.py

```diff
@@ -216,8 +220,6 @@ def stitch_curve_evenly(points: typing.Sequence[Point], stitch_length: float, to
     last = points[0]
     stitches = []
     while i is not None and i < len(points):
-        check_stop_flag()
```

Comment:
Why remove this?  Too many checks / too slow?

Reply:
This was inside a tight inner loop. I moved it out one level to be per-segment instead of per-stitch for performance.